packer/Dockerfile

```diff
@@ -0,0 +1,13 @@
+ARG ANSIBLE
+FROM quay.io/hellofresh/ci-ansible:${ANSIBLE}
```

Comment:
Why this base image? ansible/ansible on dockerhub looks abandoned, so I would probably prefer to start from ubuntu/debian and install ansible as needed

Reply:
You're right. I were fighting with a set of base images. This one was the first one that works for me.

But i agree with you; I switched it to use a debian base ;)